docs/docs/components/lifecycle-operator/webhook.md

```diff
@@ -0,0 +1,68 @@
+---
+comments: true
+---
+
+# Admission Webhooks
+
+Keptn uses
+[Admission Webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+to mutate resources.
+To enable the webhook (and therefore Keptn Lifecycle Management)
+for a certain namespace, the namespace must be annotated:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podtato-kubectl
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"  # this line tells the webhook to handle the namespace
+```
+
+The mutating webhook only modifies specifically annotated resources in the annotated namespace.
+When the webhook receives a request for a new pod,
+it looks for the workload annotations:
+
+```yaml
+keptn.sh/workload: "some-workload-name"
+```
+
+The mutation either replaces the scheduler
+with the Keptn Scheduler, or adds the
+[Scheduling Gate](https://keptn.sh/stable/docs/components/scheduling/#keptn-scheduling-gates-for-k8s-127-and-above).
+The webhook then creates a workload and app resource for each annotated resource.
+You can also specify a custom app definition with the annotation:
```

Comment:
this sentence may be confusing now that app is autogenerated.. maybe we could rephrase it

Reply:
maybe completely remove it?

Reply:
yes let's remove it it does not make sense here